api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/convolutional/AbstractConv.kt

```diff
@@ -19,6 +19,35 @@ import org.tensorflow.op.core.Variable
 import java.lang.IllegalArgumentException
 import kotlin.math.roundToInt
 
+/**
+ * Abstract Convolutional layer is a base block for building base types of convolutional layers
+ * of any dimensionality. It should simplify the internal calculations needed in most of
```

Comment:
It should simplify the internal calculations needed in most convolutional layers and abstract the naming weights for these layers.

Reply:
done